### PR TITLE
first draft  of conclusion

### PR DESCRIPTION
**Pull request by CaptainSifff**

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -492,6 +492,29 @@ Qualification badges are another venue, that RSEs to find people with the same t
...
 
+# Conclusion and Outlook
+This paper started from a community workshop at deRSE23 in Paderborn
+where people working in RSE related fields got together to figure out 
+structures and ideas of educating newcomers to this field.
+One outcome of this diverse gathering is that RSEs from far away fields gather around similar concepts.
+In this publication we have tried to formalize them. We have formulated a set of values that
+guide our actions in society and manifestly makes RSEs part of the community of values of scientists
+but at the same time, being Software Engineers, we have to take responsibility for our tools.
+We continue with core competencies that have on purpose been formulated
```
Comment (tobyhodges) [inline competencies.md:503]:
```suggestion
We continue with core competencies that have been intentionally formulated 
```

**Review by tobyhodges — APPROVED** [review]
This is looking good.

**Comment by jpthiele** [comment]
This reads really nicely

**Review by hvwaldow — COMMENTED** [review]
Include why RSEs and what they do is, nowadays, critically important for many areas of science. The proposed changes (in later papers) consitute a proposal to update the scientific ecosystem to the age of digitalization.

File: competencies.md [hunk 2]
```diff
@@ -492,6 +492,29 @@ Qualification badges are another venue, that RSEs to find people with the same t
...
+abstractly without referencing any particular information processing device.
+As expected, we draw equally upon notions from software engineering and research,
+but find that we likewise require teamwork capabilities.
+We continue with detailing these competencies in various dimensions, and find that
+a different amount is required in different positions and scientific domains.
+Nevertheless, they are required and hence the values and competencies form a common denominator
+that unifies RSEs and enables them to identify with this domain.
+These competencies at the intersection of research and software engineering 
+coupled with a firm believe in team processes makes RSEs sought after on the job market
+and their values make them responsible members of a digital society.
```
Comment (jngrad) [inline competencies.md:513]:
```suggestion
These competencies at the intersection of research and software engineering 
coupled with a firm belief in team processes makes RSEs sought after on the job market
and their values make them responsible members of a digital society.
In addition, academic institution have a growing interest to foster RSE training,
due to the increasing importance of FAIR data and FAIR software in the academic world.
Funding agencies, scientific journals and university librarians are creating new incentives
to produce openly accessible and easily findable and re-usable research artifacts.
```

Note: this should broadly cover all incentives like mandatory data repository URLs when submitting a manuscript in e.g. *Nature* journals, data sharing requirements from funding bodies like the DFG or HORIZON, and local initiatives from university libraries, who now have to deal with the archiving of scientific output that doesn't take the form of papers and dissertations. I though about ending the new paragraph with "This is achieved through digital infrastructures that require RSE skills passed down from senior researchers to newcomers as part of the onboarding process.", but it probably needs a nicer formulation that integrates with our recommendations w.r.t. teaching RSEing and skills.

File: competencies.md [hunk 3]
```diff
@@ -492,6 +492,29 @@ Qualification badges are another venue, that RSEs to find people with the same t
...
+This yields a qualification profile which makes
+an education based on them highly attractive to young people and the institutions that provide
```
[the remark this answers is not in the document]
Comment (CaptainSifff) [reply competencies.md]: fixed

File: competencies.md (hunk 2 above)
Comment (CaptainSifff) [reply competencies.md:513]: Now part of the conclusion in the main branch. Thanks!

**Pull request merged**